### ISSUE TITLE
Connect to BIE contention event streams in DEV8 env

### ISSUE THREAD

**Issue by dianagriffin** [department-of-veterans-affairs/abd-vro#1159]
**User Story**

As a VRO or CC team engineer, I want to connect VRO to the BIE kafka streams for contention events in the `DEV8` environment and document the process for doing so, so that we can validate our ability to consume contention events and have clear steps to connect in higher environments.

- [There is something about kcat but I need to find the thread and get Eng inputs to fill out this ticket]
- Premal has keys from the BIA team for the `DEV8` environment; please reach out to him to get those keys

**Acceptance Criteria**

1. Connectivity is established between VRO and the BIE contention event streams for: (1) contention created, (2) contention updated, (3) contention classified, (4) contention deleted in the `DEV8` environment

**Not included in this work**

This ticket will focus on connectivity in `DEV8`. The steps to connect in `STAGE` may be the same/very similar, but unless they can literally be accomplished at the same time, connecting in `STAGE` or any other environments should be left out of scope for this ticket. 

**Notes about work**
- [BIE kafka streams documentation](https://community.max.gov/pages/viewpage.action?spaceKey=VAExternal&title=Benefits+Integration+Events) (requires network + jira max access)

**Comment by amprokop** [comment]
Yep!

**Comment by lukey-luke** [comment]
Went to an intro chat today to get an idea of what kafka topics we should use and what the purpose of the different environments can be used for.

Should be getting VA.gov temporary access after the "transmittal notice" of my eQip. I have already looped in A6 on this

**Comment by lukey-luke** [comment]
I was successfully able to connect to a sleep pod [per lightkeeper docs](https://animated-carnival-57b3e7f5.pages.github.io/aws-iaas/#developer-access)

Looks like I need to set up a pod w/ access to kcat and I'll run through the instructions

**Comment by lukey-luke** [comment]
Specifying a different image for sleep pod also fails, so I think figure out docker auth from inside the container will be the way to solve this
```
  Warning  Failed     5s (x2 over 25s)   kubelet            Failed to pull image "nginx:1.23.3": rpc error: code = Unknown desc = Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```

**Comment by lukey-luke** [comment]
ok, after digging and failing, it looks like the k8s admission controller blocks most images, I'll see if I can use an official linux image and install kcat there

[Signed image enforcement on LHDI docs here](https://animated-carnival-57b3e7f5.pages.github.io/container-platform/signed-images/#signed-image-enforcement).

**Comment by lukey-luke** [comment]
Contacted Adrien on use of keys to correctly set up config for kafka. Pod is set up to run the test, but I either do not have the correct keys or I have added them incorrectly at this point

**Comment by yurenanava** [comment]
@lukey-luke, how can we help you to unblock this issue?

**Comment by lukey-luke** [comment]
Spoke w/ Alex this morning and he pointed me at Stephen quick, I've mentioned him on the issue in the #bip-support-ext-vro channel

**Comment by lukey-luke** [comment]
changed the way the newlines were encoded

updated the .ca to include a bunch of certificates instead of 1,

successfully completed kcat test

see slack:
https://lighthouseva.slack.com/archives/C04U8GESRT7/p1679081323505489

**Comment by lukey-luke** [comment]
Here are the corresponding topics.
(1) contention created, 
`TST_CONTENTION_BIE_CONTENTION_ASSOCIATED_TO_CLAIM_V02`

(2) contention updated, 
`TST_CONTENTION_BIE_CONTENTION_UPDATED_V02`

(3) contention classified, 
`TST_CONTENTION_BIE_CONTENTION_CLASSIFIED_V02`

(4) contention deleted in the DEV8 environment
`TST_CONTENTION_BIE_CONTENTION_DELETED_V02`



![Image](https://user-images.githubusercontent.com/17188013/227062982-514012f3-56bd-4912-8a56-8cd7681c7736.png)



![Image](https://user-images.githubusercontent.com/17188013/227062987-e897f73a-0312-4f56-a316-f2753d320f25.png)
